images/ipi-deprovision/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM registry.svc.ci.openshift.org/openshift/hive-v4.0:hive
+LABEL maintainer="hongkliu@redhat.com"
+
+COPY ipi-deprovision/ipi-deprovision.sh  /usr/bin/ipi-deprovision.sh
+
+# Install awscli
+RUN chmod +x /usr/bin/ipi-deprovision.sh && \
+      curl -O https://bootstrap.pypa.io/get-pip.py && \
+      python get-pip.py && \
+      rm -fv get-pip.py && \
+      pip install awscli
```

Comment:
I'm wondering whether we should peg this to a specific version to have control over what we are building upon.

Reply:
Done.